includes/Discovery.php

```diff
@@ -70,6 +62,17 @@ static function () {
 		add_action( 'web_stories_story_head', 'wp_oembed_add_discovery_links' );
 	}
 
+	/**
+	 * Theme support for title-tag is implied for stories. See _wp_render_title_tag().
+	 *
+	 * @return void
+	 */
+	public function print_title_metadata() {
```

Comment:
Can you rename this to `print_metadata`? #2336

I don't want a method for every single field. `print_metadata` for title & description sounds good enough.

```suggestion
	public function print_metadata() {
```